x/stablestake/types/codec.go

```diff
@@ -3,15 +3,23 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
-	// this line is used by starport scaffolding # 1
```

Comment:
@jelysn This might break Ignite CLI. Have you tested Ignite CLI is still working on this branch?

Reply:
There was no plan to use ignite scaffolding tool anymore on this module. Rest of the stuff can be done with handwriting. Let me revert this change though

Reply:
https://github.com/elys-network/elys/pull/207/commits/888857ad18570ffa2fc385efa55f056fe63f07b2
Done